static/src/javascripts/projects/common/utils/url.js

```diff
@@ -1,8 +1,7 @@
 define([
     'common/utils/detect',
-    'common/utils/mediator',
-    'lodash/utilities/identity'
-], function (detect, mediator, identity) {
+    'common/utils/mediator'
```

Comment:
unrelated to your changes, but I was intrigued by a dependency on `mediator`. I think it's a wrong design decision for a utility to listen to events. I tried to search for the subscribed event and found nothing. Wondering if that push state still used?